spec/features/workflow/unschedule_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+RSpec.feature "Unschedule an edition" do
+  scenario do
+    given_there_is_a_scheduled_document
+    when_i_visit_the_summary_page
+    and_i_click_stop_scheduled_publishing
+    then_the_document_is_no_longer_scheduled_and_the_status_reverted
+    and_the_scheduled_datetime_is_still_persisted
+  end
+
+  def given_there_is_a_scheduled_document
+    schedule = create(:scheduling)
+    @datetime = Time.current.tomorrow.at_noon
+    @edition = create(:edition,
+                      :scheduled,
+                      scheduling: schedule,
+                      scheduled_publishing_datetime: @datetime)
+  end
+
+  def when_i_visit_the_summary_page
+    visit document_path(@edition.document)
+  end
+
+  def and_i_click_stop_scheduled_publishing
+    click_on "Stop scheduled publishing"
+  end
+
+  def then_the_document_is_no_longer_scheduled_and_the_status_reverted
+    expect(@edition.status.details.pre_scheduled_status.draft?).to eq(true)
+    expect(@edition.scheduled?).to eq(true)
+
+    @edition.reload
+
+    expect(@edition.draft?).to eq(true)
+    expect(@edition.timeline_entries.last.entry_type).to eq("unscheduled")
+  end
+
+  def and_the_scheduled_datetime_is_still_persisted
+    expect(@edition.scheduled_publishing_datetime).to eq(@datetime)
```

Comment:
If Sonia decides we should still show the set scheduled_publishing_Datetime in the form, we should check that it's still displayed in the UI rather than in the db.

Reply:
Good point, after a chat with Sonia and how the new design works we decided it would be best not to save the time after cancelling. 

We could always amend this if it was seen to be a problem later on.